src/main/java/cotato/backend/global/common/ApiResponse.java

```diff
@@ -0,0 +1,39 @@
+package cotato.backend.global.common;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+
+import java.time.LocalDateTime;
+
+@Getter
+@AllArgsConstructor
+@Builder
```

Comment:
```suggestion
@Builder
@JsonInclude(JsonInclude.Include.NON_NULL) 
```
해당 어노테이션을 사용하면 null 값은 응답에 포함하지 않게 구성이 가능합니다! 추가해주는것도 좋을 것 같아요!

Reply:
피드백 주신 내용 반영 완료했습니다! 
다시 한 번 확인 부탁드립니다 :)